plugins/services/src/js/constants/NewApplicationDefaults.js

```diff
@@ -0,0 +1,7 @@
+const NEW_APP_DEFAULTS = {
+  cpus: 0.001,
+  instances: 1,
+  mem: 128
+};
+
+module.exports = NEW_APP_DEFAULTS;
```

Comment:
I am in favour of this, but we are kind of breaking the coding pattern doing so. Perhaps we could run this through the team?